src/PKSim.Core/Snapshots/Simulation.cs

```diff
@@ -45,5 +46,20 @@ public IReadOnlyList<Chart> Analyses
             return new List<Chart>();
          }
       }
+
+      public LocalizedParameter ParameterByPath(string parameterPath) =>
+         Parameters?.Find(x => string.Equals(x.Path, parameterPath));
+
+      public void AddOrUpdate(LocalizedParameter parameter)
+      {
+         var existingParameter = ParameterByPath(parameter.Path);
+         var localizedParameters = new List<LocalizedParameter>(Parameters ?? Enumerable.Empty<LocalizedParameter>());
+         if (existingParameter != null)
+            localizedParameters.Remove(existingParameter);
+
+         localizedParameters.Add(parameter);
+
+         Parameters = localizedParameters.ToArray();
```

Comment:
Dealing with an array is great for json import and export but not so great when adding removing! easy enough however